pages.de/linux/nordvpn.md

```diff
@@ -0,0 +1,24 @@
+# nordvpn
+
+> Kommandozeilen-Schnittstelle für NordVPN.
+> Weitere Informationen: <https://nordvpn.com/download/linux/>.
+
+- Interaktiv bei einem NordVPN-Konto anmelden:
+
+`nordvpn login`
+
+- Zeige den Verbindungsstatus an:
+
+`nordvpn status`
+
+- Verbinden dem nächsten NordVPN-Server:
```

Comment:
In German it's often tricky to use the imperative form, while not using personal pronouns. What do you think about
```suggestion
- Stelle eine Verbindung zum nächsten NordVPN-Server her:
```